packages/hyperverse-evm-mintpass/contracts/MintPass.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import './hyperverse/IHyperverseModule.sol';
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract MintPass is ERC1155, Ownable, IHyperverseModule {
+	string public _name;
+  	string public _symbol;
+	
+	uint256 public tokenCounter;
+	bool public initialized;
```

Comment:
we can get rid of this initialized variable if we just use a modifier to check 

reference:
https://github.com/decentology/hyperverse-mono/blob/099dc7b340f07b4d3abbd3b675fa29952ff4b977/packages/hyperverse-evm-whitelist/contracts/Whitelist.sol#L87